doc/manual.xml.head

```diff
@@ -7304,6 +7304,14 @@ set index_format="%4C %Z %{%b %d} %-15.15L (%?l?%4l&amp;%4c?)%*  %s"
                   <quote>Message-ID</quote> field
                 </entry>
               </row>
+              <row>
+                <entry>~I <emphasis>QUERY</emphasis></entry>
+                <entry>
+                  messages whose <quote>Message-ID</quote> field is included in
+                  the results returned from en external search program, when the program
+                  is run with <emphasis>QUERY</emphasis> as its argument. *****)
```

Comment:
Is `*****)` intended?

Reply:
Yes, that's what I intended.  It refers to a footnote later on.  But maybe I should just inline the text I put in the footnote, especially because it consits of a further cross-reference.  What do you think?

Reply:
Oh, I'm fine either way - just wasn't aware of the notation.